docs/getting-started-with-react/getting-started-with-react.stories.mdx

```diff
@@ -154,10 +154,10 @@ The next section is the return value which will return the TSX defining our
 component. In here you can see that our `GifGift` component is composed of the
 following other react components:
 
-- `Card` — [Docs](/components/card)
-- `Content` — [Docs](/components/content)
-- `Button` — [Docs](/components/button)
-- `InputText` — [Docs](/components/input-text)
+- `Card` — [Docs](../?path=/docs/components-card--card)
```

Comment:
I know this is how the original docs had it but after going through everything, this list looks a bit out of place with having ` - Docs` at the end. Maybe take away the backticks and just have this? Up to you...
```suggestion
- [Card](../?path=/docs/components-card--card)
```

Reply:
I like this! 👍